src/reducers/entities/channels.ts

```diff
@@ -389,7 +377,7 @@ function myMembers(state: RelationOneToOne<Channel, ChannelMembership> = {}, act
         if (sync) {
             current.forEach((member: ChannelMembership) => {
                 const id = member.channel_id;
-                if (!channelMembers.find((cm: ChannelMembership) => cm.channel_id === id)) {
+                if (channelMembers.find((cm: ChannelMembership) => cm.channel_id === id)) {
```

Comment:
this accomplishes the same thing as `remove` in the `RECEIVED_MY_CHANNEL_MEMBERS` action